R/project.R

```diff
@@ -18,17 +18,24 @@
 #'
 #' }
 project <- function(default = NULL) {
-  renv_project(default = default)
+  renv_project_get(default = default)
 }
 
-renv_project_get <- function(default = NULL) {
-
+renv_project_get <- function(default = getwd()) {
   project <- Sys.getenv("RENV_PROJECT", unset = NA)
   if (is.na(project))
     return(default)
 
   project
+}
+
+renv_project_resolve <- function(project = NULL) {
+  project <- project %||% renv_project_get()
```

Comment:
```suggestion
  project <- project %??% renv_project_get()
```

Reply:
I'm not sure you want to get into the details, but why have a separate `%??%` and `%||%` and why make `%??%` more like the `%||%` we use everywhere else? 😭